SimulationAgent/DeviceState/DeviceStateActor.cs

```diff
@@ -31,6 +32,13 @@ public enum ActorStatus
         /// periodically updated using an external script.
         /// </summary>
         public Dictionary<string, object> DeviceState { get; set; }
+        public bool IsDeviceActive
+        {
+            get
+            {
+                return this.status == ActorStatus.Updating;
```

Comment:
Curious, why is device active state same as updating ?

Reply:
this is by design, basically rather than talking about "connected devices" we use the term "active devices" which is a bit more generic. So when we show the number of active devices, we can include devices which are not connected yet but being simulated.

@maple10001 please add this text to the comment above